contrib/gitian/README.md

```diff
@@ -201,10 +235,12 @@ Make a pull request (both the `.assert` and `.assert.sig` files) to the
 [monero-project/gitian.sigs](https://github.com/monero-project/gitian.sigs/) repository:
 
 ```bash
+git clone https://github.com/$GH_USER/gitian.sigs.git
```

Comment:
This clone step is redundant, since it was already done at line 158 above.

Reply:
Yeah. Removed. I'm still trying to embrace your workflow.